Legacy 2 - update po live Q&A (#1212)
tpwres/teczki-polskiego-wrestlingu: content/o/low.md

### 2025: The second show

On 27.01.2025 Legacy [published a teaser image][low-2-teaser] for "Legacy of Wrestling Vol. 2". A [trailer][low-2-trailer] for the show was published on 18.02.2025, along with an announcement of [another live Q&A][live-qa-3], which took place on 19.02.2025. Again, the Q&A was hosted by [ShowOff](@/w/piotr-malecki.md).

* The show is named "Legacy of Wrestling vol. 2" in reference to [Guardians of the Galaxy Vol. 2][galaktyczni-2].

> * Like previously, the show will take place on a Sunday, as this was the only available date that didn't clash with shows by other promotions.

Reviewer: ```suggestion
* Like previously, the show will take place on a Sunday, as this was the only available date that didn't clash with other wrestling shows.
```

* The chances for someone from [Prime Time Wrestling](@/o/ptw.md) appearing at the show are nearly zero, with the current LOW-PTW relations being "complicated".
* Events from [KPW](@/o/kpw.md), [MZW](@/o/mzw.md) and [PpW](@/o/ppw.md) shows are considered canon by Legacy, but what happens at Legacy shows may or may not be considered canon by other promotions.
* There will be fewer seats than the last time in order to fit more people in the venue, and sell more tickets.
* There will be no Patreon this time around.
* The limited T-shirts should be available for sale at the show, but it'll be cheaper to buy the VIP ticket with the T-shirt included.
* People who don't train as much will be used in roles other than in-ring competition, to keep the quality of the show consistently high.
* Legacy are considering their own championship titles, but first they want to establish themselves as a promotion with regular events.
* The second show will air on Extreme Sports Channel, and will be made available on YouTube about two weeks later.

> * There are talks with regards to international wrestlers. Legacy are willing to have one or two guests from abroad, but Polish wreslers are their primary focus.

Reviewer: ```suggestion
* There are talks with regards to international wrestlers. Legacy are willing to have one or two guests from abroad, but Polish wrestlers are their primary focus.
```

> * Talks with KPW before the first show started with the federation, before moving to individual wreslers.

Reviewer: ```suggestion
* Prior to the first show, Legacy talked with KPW first, before moving on to their individual wrestlers.
```

* Legacy hope to have some KPW wrestlers at the second event as well.
* The ring will most likely be supplied by MZW again.
* The musical chairs from the first show were a reference to a [similar segment][krzesla-balora] in Progress Wrestling, featuring Finn Bálor.
* There were no talks with [Mine City Wrestling](@/o/mcw.md) this time; Legacy don't see a feasible way to have MCW be a part of the show with both sides being satisfied.
* Legacy talked to MCW with the hopes of borrowing their ring for the first show, but those talks fell through.
* There's a chance Express Pizza might be Legacy's sponsor again.
* Kubes, [Dziedzic](@/w/dziedzic.md), [Balik](@/w/lukasz-balinski.md), [Sambor](@/w/sambor.md) and Mister Chairman were confirmed to be at the second show (though Mr. Chairman may or may not appear publicly).
* [Arek Paterek](@/w/arek-paterek.md) should be at the show too; [Disco Pablo](@/w/disco-pablo.md) perhaps as well.
* Everyone who appeared at the first show is invited to the second one.

> * There will be matches between wreslers from different promotions.

Reviewer: ```suggestion
* There will be matches between wrestlers from different promotions.
```
